src/renderer.ts

```diff
@@ -517,6 +517,33 @@ const render = (format: Format, excludeStyles = false) =>
             })
         }
 
+        case ElementKind.ProfileAtom: {
+            return h(ProfileAtom, {
+                ...element,
+                pillar: pillarToString(format.pillar),
+                likeHandler: () => { console.log("like clicked"); },
+                dislikeHandler: () => { console.log("dislike clicked"); },
+                expandCallback: () => { console.log("expand clicked"); }
+            })
+        }
+
+        case ElementKind.TimelineAtom: {
+            return h(TimelineAtom, {
+                ...element,
+                pillar: pillarToString(format.pillar),
+                likeHandler: () => { console.log("like clicked"); },
+                dislikeHandler: () => { console.log("dislike clicked"); },
+                expandCallback: () => { console.log("expand clicked"); }
+            })
+        }
+
+        case ElementKind.ChartAtom: {
+            return h(ChartAtom, {
+                ...element,
+                url: ""
```

Comment:
What's the `url` field meant to be used for?

Reply:
It's actually not used, I'll try to make a PR to atoms-rendering.

Reply:
It's already been updated, let me bump atoms-rendering